code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/EdgePersonalizationResponseHandler.java

```diff
@@ -392,17 +402,10 @@ void retrieveMessages(final List<Surface> surfaces, final Event event) {
             return;
         }
 
-        final Map<Surface, List<Proposition>> ruleConsequencePropositions =
-                getPropositionsFromFeedRulesEngine(event);
-        Map<Surface, List<Proposition>> requestedPropositions =
-                retrieveCachedPropositions(requestedSurfaces);
```

Comment:
Removing this breaks `getPropositionsForSurfaces` for CBE. Cached CBE propositions are never returned